Tests/FluentAssertions.Json.Specs/JTokenAssertionsSpecs.cs

```diff
@@ -445,6 +446,52 @@ public void When_a_token_is_equal_to_its_string_representation_the_non_equivalen
                 .WithMessage("Expected JSON document not to be equivalent*");
         }
 
+        [Fact]
+        public void When_a_float_is_whithin_approximation_check_should_succeed()
+        {
+            //-----------------------------------------------------------------------------------------------------------
+            // Arrange
+            //-----------------------------------------------------------------------------------------------------------
+            var actual = JToken.Parse("{ \"id\": 1.1232 }");
+            var expected = JToken.Parse("{ \"id\": 1.1235 }");
+
+            //-----------------------------------------------------------------------------------------------------------
+            // Act & Assert
+            //-----------------------------------------------------------------------------------------------------------
+            const double precision = 1e-3;
+            var options = new EquivalencyAssertionOptions<object>();
+            options.Using<double>(d => d.Subject.Should().BeApproximately(d.Expectation, precision))
+                .WhenTypeIs<double>();
+            actual.Should().BeEquivalentTo(expected, AssertionOptions => options);           
+        }
+        [Fact]
+        public void When_a_float_is_not_whithin_approximation_check_should_throw()
+        {
+            //-----------------------------------------------------------------------------------------------------------
+            // Arrange
+            //-----------------------------------------------------------------------------------------------------------
+            var actual = JToken.Parse("{ \"id\": 1.1232 }");
+            var expected = JToken.Parse("{ \"id\": 1.1235 }");
+
+            //-----------------------------------------------------------------------------------------------------------
+            // Act & Assert
+            //-----------------------------------------------------------------------------------------------------------
+            const double precision = 1e-5;
+            var options = new EquivalencyAssertionOptions<object>();
+            options.Using<double>(d => d.Subject.Should().BeApproximately(d.Expectation, precision))
+                .WhenTypeIs<double>();
+            var expectedMessage =
+                "JSON document has a different value at $.id." +
+                "Actual document" +
+                $"{Format(actual, true)}" +
```

Comment:
🤔 I would not use `Format` here. In fact, I would only assert the things which are specific to this case, and then only the parts you need to verify the approximate assertion was correctly executed. In other words, always use wildcards in `WithMessage`